internal/cnpi/plugin/client/lifecycle.go

```diff
@@ -34,15 +35,10 @@ import (
 	"github.com/cloudnative-pg/cloudnative-pg/pkg/management/log"
 )
 
-var runtimeScheme = runtime.NewScheme()
-
-func init() {
-	_ = scheme.AddToScheme(runtimeScheme)
-}
-
-func (data *data) LifecycleHook(
+func (data *data) LifecycleHook( //nolint:gocognit
 	ctx context.Context,
 	operationType plugin.OperationVerb,
+	schema *runtime.Scheme,
```

Comment:
I feel it could be passed through a builder or something instead of adding one more parameter.